src/components/SearchGridFilter.vue

```diff
@@ -112,6 +112,7 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import "@creativecommons/vocabulary/scss/spacing.scss";
```

Comment:
You don't have to import the .scss file to use the variables. You can use the `margin` and `padding` classes in the appropriate element. See https://cc-vocabulary.netlify.com/?path=/docs/tokens-spacing--figma#usage

Reply:
Done @brenoferreira